lib/ontologies_linked_data/utils/notifications.rb

```diff
@@ -209,37 +146,71 @@ def self.mail_options
 %note_body%<br/>
 ----------------------------------------------------------------------------------<br/><br/>
 
-You can respond by visiting: <a href="%note_url%">NCBO BioPortal</a>.<br/><br/>
+You can respond by visiting: <a href="%note_url%">%ui_name%</a>.<br/><br/>
 EOS
 
-SUBMISSION_PROCESSED = <<EOS
-%ontology_name% (%ontology_acronym%) was processed for use in BioPortal. Here are the results:
+      SUBMISSION_PROCESSED = <<EOS
+%ontology_name% (%ontology_acronym%) was processed for use in %ui_name%. Here are the results:
 <br><br>
 %statuses%
 <br><br>
 Please contact %admin_email% if you have questions.
 <br><br>
-The ontology can be <a href="%ontology_location%">browsed in BioPortal</a>.
+The ontology can be <a href="%ontology_location%">browsed in %ui_name%</a>.
 <br><br>
 Thank you,<br>
-The BioPortal Team
+The %ui_name% Team
 EOS
 
-REMOTE_PULL_FAILURE = <<EOS
-BioPortal failed to load %ont_name% (%ont_acronym%) from URL: %ont_pull_location%.
+      REMOTE_PULL_FAILURE = <<EOS
+%ui_name% failed to load %ont_name% (%ont_acronym%) from URL: %ont_pull_location%.
 <br><br>
 Please verify the URL you provided for daily loading of your ontology:
 <ol>
-<li>Make sure you are signed in to BioPortal.</li>
-<li>Navigate to your ontology summary page: %ontology_location%.</li>
+<li>Make sure you are signed in to %ui_name%.</li>
+<li>Navigate to your ontology summary page: <a href="%ontology_location%">%ontology_location%</a>.</li>
 <li>Click the &quot;Edit submission information&quot; link.</li>
 <li>In the Location row, verify that you entered a valid URL for daily loading of your ontology in the URL text area.</li>
 </ol>
-If you need further assistance, please <a href="mailto:support@bioontology.org">contact us</a> via the BioPortal support mailing list.
+If you need further assistance, please <a href="mailto:%support_mail%">contact us</a> via the %ui_name% support mailing list.
```

Comment:
This PR introduces a new configuration variable `LinkedData.settings.admin_emails` for sending notifications to OntoPortal Administrators which is needed for replacing hard-coded email address.

If this PR gets deployed in BioPortal then users will start geting emails asking them to contact administrators of BioPortal instead of the support email list. 
Perhaps we need to introduce a second configuration option to distinguish contacting support vs contacting admin.